### ISSUE TITLE
`%%capture` does not captures `stderr`

### ISSUE THREAD

**Issue by pya** [jupyterlab/jupyterlab#11775]
## Description

`%%capture` does not captures `stderr`

## Reproduce

Run a cell with `%%capture` and raise an exception:

```python

%%capture cap
1 / 0
---------------------------------------------------------------------------
ZeroDivisionError                         Traceback (most recent call last)
/var/folders/p_/yckgs9wn0xbcmqc92zk3z4240000gn/T/ipykernel_7816/2381044780.py in <module>
----> 1 1 / 0

ZeroDivisionError: division by zero
```

and `stderr` on the capture object is empty.

```python
cap.stderr
''
```

## Expected behavior

I expect the exception to be suppress and the exception text written to the attribute `stderr` of the capture object.

## Context

Python: 3.10.1 | packaged by conda-forge | (main, Dec 22 2021, 01:39:14) [Clang 11.1.0 ]
JupyterLab: Version 3.2.5
IPython:  7.30.1
Platform: MacOS 10.14
Browser: Brave (Chromium based, Version 1.33.106 Chromium: 96.0.4664.110(x86_64))
and Firefox 95.0.2 (64-bit)

## Related

Looks like there is a bug in IPython that might be related:
https://github.com/ipython/ipython/issues/13425

**Comment by krassowski** [comment]
Hi @pya thank you for opening this issue. I believe that it is a behaviour of IPython; JupyterLab does not have a code related to it, so we cannot address it in here. I will explain why it works like it over in https://github.com/ipython/ipython/issues/13425.

**Comment by pya** [comment]
Solved here:  ipython/ipython#13425.
